docs/features/menu-legend-links.md

```diff
@@ -0,0 +1,37 @@
+# Menu/legend info Links
+gisida-react menu component builds menu items/sections based on json spec provided on clients site-config file. Menu/legend info links feature seeks to give context of the selected layer, menu sections/subsections. The links can be added by adding links and descriptions to layer/siteconfig as shown below.    
+
+## Usage and expected behaviour
+
+The following are configs to be added when implementing this feature.
+
+### Site level menu (sector/subsector)
+##### Sector level spec
+The `link` and `description` fields are optional one can be provided another ommitted as desired
+```
+"hyperLink": {
+      "Boundaries": {
+        "description": "desc",
+        "link": "https://gisida.com",
+      }
+    },
+
+```
+#### Sub-Sector level spec
+The `parent category` is used to identify the sub-sector based on parent sector/sub-sector. The parent category should be appended before sector/sub-sector key inside hyperlink property. This helps when we have same sub-sector names in different sectors.
+
+BRCis II is the parent in this case
+```
+"BRCiS II Output Indicators": {
+        "link": "https://gisida.com",
+        "description": "desc"
+      }
+```
+#### Legend (layer level) spec
```

Comment:
Could you please add a layer spec including these new properties so it's a bit clear where to add them in the layer spec?